resources/user-agents/browsers/maxthon/maxthon-nitro-1-0-on.json

```diff
@@ -9,7 +9,7 @@
       "userAgent": "Maxthon Nitro #MAJORVER#.#MINORVER#",
       "platform": "Windows",
       "engine": "Blink",
-      "device": "general Desktop",
+      "device": "Windows Desktop",
```

Comment:
This was just another I noticed while clicking around. There are no other children in this file, so defaulting to `Windows Desktop`.